### PR TITLE
CAN: support sending actuators as PWM

### PR DESCRIPTION
**Pull request by tridge**
This makes debugging CAN servo setups easier
image

### CODE REVIEW

File: Tools/AP_Periph/rc_out.cpp [hunk 1]
```diff
@@ -109,6 +109,16 @@ void AP_Periph_FW::rcout_srv(uint8_t actuator_id, const float command_value)
 #endif
 }
 
+void AP_Periph_FW::rcout_srv_PWM(uint8_t actuator_id, const float command_value)
+{
+#if HAL_PWM_COUNT > 0
+    const SRV_Channel::Aux_servo_function_t function = SRV_Channel::Aux_servo_function_t(SRV_Channel::k_rcin1 + actuator_id - 1);
+    SRV_Channels::set_output_pwm(function, uint16_t(command_value+0.5));
```
Comment (andyp1per) [inline Tools/AP_Periph/rc_out.cpp:116]: What is the magic 0.5 for?

Comment (IamPete1) [reply Tools/AP_Periph/rc_out.cpp:116]: Rounding float to int I guess.

Comment (tridge) [reply Tools/AP_Periph/rc_out.cpp:116]: yep

**Review by IamPete1 — APPROVED** [review]
LGTM

**Pull request merged**